config/default.settings.local.php

```diff
@@ -13,18 +13,18 @@
  */
 
 // Database settings.
-$databases = array (
+$databases = array(
   'default' =>
-    array (
+    array(
       'default' =>
-        array (
+        array(
           'database' => 'dosomething',
           'username' => 'root',
           'password' => '',
-          'host' => 'localhost',
-          'port' => '',
-          'driver' => 'mysql',
-          'prefix' => '',
+          'host'     => 'localhost',
```

Comment:
I'm not super enthusiastic about the `align_double_arrows` fixer, since it means potentially mangling diffs by having to re-align other array items if you add a key that "pushes" the other arrows over.